hillel_hw_6/iterator_protocol.py

```diff
@@ -0,0 +1,33 @@
+# 1. Створити frange ітератор. Який буде працювати з float.
+
+class frange:
+    def __init__(self, start, stop=None, step=1.0):
+        self.start = start
+        self.step = step
+        self.stop = stop
+        if self.stop is None:
+            self.start, self.stop = 0.0, self.start
+
+    def __iter__(self):
```

Comment:
this is a generator, and you need to create an iterator.

Add iter and next methods.